formsfeeder.client/src/main/java/formsfeeder/client/support/BuilderImpl.java

```diff
@@ -74,13 +79,25 @@ public BuilderImpl basicAuthentication(String username, String password) {
 	}
 
 	@Override
-	public BuilderImpl addQueryParam(String name, Supplier<Object> value) {
+	public BuilderImpl addQueryParam(String name, List<Supplier<String>> value) {
 		this.queryParams.put(name,value);
 		return this;
 	}
 
 	@Override
-	public Map getQueryParams() { return this.queryParams; }
+	public BuilderImpl addQueryParam(String name, Supplier<String> value) {
+		if(this.queryParams.containsKey(name)) {
+			this.queryParams.get(name).add(value);
+		} else {
+			List<Supplier<String>> list = new ArrayList<>();
+			list.add(value);
+			this.queryParams.put(name, list);
+		}
+		return this;
+	}
+

```

Comment:
When I run code coverage on the this project, I see that this routine is missed completely.  It looks like we're always adding Lists of Suppliers and never adding Suppliers by themselves.  Would you mind either altering some of the existing tests or adding new tests to cover both code paths in this routine?